src/Cms/PageSiblings.php

```diff
@@ -16,101 +16,75 @@ trait PageSiblings
 	/**
 	 * Checks if there's a next listed
 	 * page in the siblings collection
-	 *
-	 * @param \Kirby\Cms\Collection|null $collection
 	 */
-	public function hasNextListed($collection = null): bool
+	public function hasNextListed(Collection $collection = null): bool
```

Comment:
I think this and the following should be:

```suggestion
	public function hasNextListed(Collection|null $collection = null): bool
```

Reply:
I only didn't do it cause https://github.com/getkirby/kirby/pull/6377 will add those all automatically everywhere needed.